### DISCUSSION TITLE
posting new photo; caption entry

### DISCUSSION THREAD

**Discussion by theotherlinh** [pixelfed/pixelfed-rn#3] [Bug Reports]
In iOS, if you pick the caption field before adding anything, there's no way to go back without discarding the post. Rare, but there are times I've written things first.

Somewhat related, if you click in the caption, don't type anything, and back out, you'll be put back in the same predicament of forced text entry w/o access to the attachment options. You have to type something to trigger the back out-discard prompt.

**Comment by dansup** [comment]
This isn't applicable to the new app!
